.gitignore

```diff
@@ -8,7 +8,7 @@ tools/bcdb_mock/bcdb_mock
 tools/tfuser/tfuser
 tools/tffarmer/tffarmer
 tools/schemac/schemac
-tools/upgradectl/upgradectl
+tools/updatectl/updatectl
```

Comment:
I don't think this needs to be renamed

Reply:
I didn't rename it, i just added the correct name to .gitignore